app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'CMSFilterApp'
+
+app = Chalice(app_name=app_name)
+
+# Initialize the logger
+logging.basicConfig(filename='app.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(app_name)
+
+# Set the log level from Environment variable LOG_LEVEL
+if os.environ.get('LOG_LEVEL') is None:
+    logger.setLevel(logging.DEBUG)
+else:
+    logger.setLevel(os.environ['LOG_LEVEL'])
+
+kinesis_endpoint_url = os.environ.get('KINESIS_ENDPOINT_URL')
+aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID', 'iam')
+aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY', 'iam')
+aws_region = os.environ.get('AWS_REGION', 'us-east-1')
+output_stream_name = os.environ.get('OUTPUT_STREAM')
+logger.debug('##Environment variables {}'.format(os.environ))
```

Comment:
do we actually need that kinesis urls? since the lambda is stitched with a specific stream, the default aws creds are probably enough.

Reply:
I think yes here we need the URL for testing it with the local stack, where I need to pass the local stack endpoint.